pandas/util/testing.py

```diff
@@ -1400,7 +1400,9 @@ def assert_frame_equal(
                 check_names=check_names,
                 check_datetimelike_compat=check_datetimelike_compat,
                 check_categorical=check_categorical,
-                obj="{obj}.iloc[:, {idx}]".format(obj=obj, idx=i),
+                obj='{obj}.iloc[:, {idx}] (column name="{col}")'.format(
```

Comment:
If you wanted to could just use f-strings to make this even more compact

Reply:
They are new in 3.6. Is anything older not supported anymore by pandas?

Reply:
no we just dropped 3.5